Purchases/Misc/RCCrossPlatformSupport.h

```diff
@@ -37,25 +37,23 @@
 
 // Should match available platforms in
 // https://developer.apple.com/documentation/uikit/uidevice?language=objc
-#if TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_MACCATALYST
+#if TARGET_OS_IOS || TARGET_OS_TV
 #define UI_DEVICE_AVAILABLE 1
 #else
 #define UI_DEVICE_AVAILABLE 0
 #endif
 
 // Should match available platforms in
 // https://developer.apple.com/documentation/iad/adclient?language=objc
-#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+#if TARGET_OS_IOS
```

Comment:
so `TARGET_OS_MACCATALYST` is under `TARGET_OS_IOS`?

Reply:
yep

Reply:
that's what caused https://github.com/RevenueCat/purchases-ios/pull/247 as well